src/pages/RequestSchoolProducts/RequestSchoolProducts.module.scss

```diff
@@ -0,0 +1,20 @@
+@use '@assets/global.scss' as g;
+
+.container {
+  color: g.$primary-dark-blue;
+  padding: 50px;
+  .subheading {
```

Comment:
Can we add whitespace between each of these. 

E.g. 
padding: 50px;

.subheading {
....